lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/exploring-covid.adoc

```diff
@@ -0,0 +1,67 @@
+= Exploring the Covid Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to have the @starter-file{alg2-covid} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `covid-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a,1a", frame="none", stripes="none"]
+|===
+| *state* - the state reporting the data
+| *day* - the number of days after June 20th, 2020
+| *positive* - the number of new, positive cases reported
+|===
+
+@n Evaluate `CT1`, `MA1` and `ME1` in the Interactions Area. Which state(s) and day(s) are represented by these rows? @fitb{}{@ifsoln{CT, MA, and ME - day 1}}
+
```

Comment:
Not sure this is the most useful/interesting question to ask here, but there definitely isn't room for students to answer it.